src/Patches/GhostHints.cs

```diff
@@ -332,23 +332,33 @@ public HintGhost() { }
-                new HintGhost("Hint Ghost Library Exterior 2", "Library Exterior", new Vector3(-8.2745f, 44.0645f, -36.2473f), new Quaternion(0.018f, -0.2001f, -0.0037f, -0.9796f), NPC.NPCAnimState.SIT, $"#uh lIbrArEuhn iz uhp #Ar... $oud bE A tehkstbouk\nfIt for yoo #O!") }
+            { "Library Rotunda", new List<HintGhost>() {
+                new HintGhost("Hint Ghost Library Rotunda 1", "Library Rotunda", new Vector3(-25.3f, -8.2f, 38.7f), new Quaternion(0f, .9343f, 0f, -.3566f), NPC.NPCAnimState.PRAY, $"wI did #uh lIbrArEuhn Evehn bild awl #Ez lahdurz?\nkahnt hE flI?"),
+                new HintGhost("Hint Ghost Library Rotunda 2", "Library Rotunda", new Vector3(-8.5f, -7f, 1.5f), new Quaternion(0f, .9973f, 0f, .0738f), NPC.NPCAnimState.SIT, $"#is yoosd too bE uh bAbE slorm rAstrahk bahk in %uh dA,\ngood tImz...") }
             },
-            { "Dusty", new List<HintGhost>() {
-                new HintGhost("Hint Ghost Dusty", "Dusty", new Vector3(64.3287f, 26.058f, 46.5875f), new Quaternion(0f, 0.9282f, 0f, -0.372f), NPC.NPCAnimState.IDLE, $"R yoo goi^ too klEn uhp #is mehs? wuht A rilEf!", cameraYOverride: 50), }
+            { "Library Lab", new List<HintGhost>() {
+                new HintGhost("Hint Ghost Library Lab 1", "Library Lab", new Vector3(139.3969f, 93.5073f, -74.8239f), new Quaternion(0f, 1f, 0f, 0.0089f), NPC.NPCAnimState.SIT, $"Im #uh lIbrArEuhn!---... juhst kidi^, I kahnt rEd."),
+                new HintGhost("Hint Ghost Library Lab 2", "Library Lab", new Vector3(131.3964f, 107.4616f, -37.7488f), new Quaternion(0f, 0.7272f, 0f, 0.6864f), NPC.NPCAnimState.FISHING, $"awl #is rEsurj ahnd #A stil dOnt nO wuht A\ndE pahd iz..."),
+                new HintGhost("Hint Ghost Library Lab 3", "Library Lab", new Vector3(145.05f, 87f, -111.35f), new Quaternion(0f, .8829f, 0f, -.4695f), NPC.NPCAnimState.SIT, $"O, yoo wawnt too nO wuht Im rEdi^? itz kawld\nfawks bEhAvyur: intrO too yahpi^ ehnd %EvurE.") }
             },
             { "Cathedral", new List<HintGhost>() {
-                new HintGhost("Hint Ghost Cathedral", "Cathedral Redux", new Vector3(-14.5132f, -0.2167f, -64.2287f), new Quaternion(0f, 0.9629f, 0f, -0.2699f), NPC.NPCAnimState.IDLE, $"hahv yoo hurd uhbowt owur lord ahnd sAvyur ahndroo $OldIs?"), }
+                new HintGhost("Hint Ghost Cathedral", "Cathedral Redux", new Vector3(-14.5132f, -0.2167f, -64.2287f), new Quaternion(0f, 0.9629f, 0f, -0.2699f), NPC.NPCAnimState.IDLE, $"hahv yoo hurd uhbowt owur lord ahnd sAvyur ahndroo $OldIs?"),
+                new HintGhost("Hint Ghost Cathedral 2", "Cathedral Redux", new Vector3(-10.9f, 1.45f, -65.2f), new Quaternion(0f, .9238f, 0f, -.3829f), NPC.NPCAnimState.SIT, $"hI, Im #uh wuhn hoo rEsehts awl uhv #uh buhtehn wawlz.") }
```

Comment:
```suggestion
                new HintGhost("Hint Ghost Cathedral 2", "Cathedral Redux", new Vector3(-10.9f, 1.45f, -65.2f), new Quaternion(0f, .9238f, 0f, -.3829f), NPC.NPCAnimState.SIT, $"hI, Im #uh wuhn hoo rEsehts awl uhv #uh buhtuhn wawlz.") }
```

Reply:
Maybe `buhtin`?

Reply:
`buhtin` is how I would say it but apparently `butuhn` is correct? idk words are hard lol

Reply:
eh, either works -- let's go with `buhtin` since we both pretty much agreed on it and since no one's really gonna care if it's slightly wrong